.github/workflows/testing.yml

```diff
@@ -64,6 +64,11 @@ jobs:
             python: 3.8
             toxenv: py38
 
+          - name: Documentation
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: build_docs
```

Comment:
This is really straightforward code.  I thought it was going to be a lot more complicated.  Nice!

Reply:
Admittedly I'm not saving any artifacts from this, just having this scream at us about warnings! RTD does the doc output now, so there's no reason to duplicate that.

Reply:
Transitioning to `tox` a while ago paid off!